changelog.d/19370_remove_warning_for_unused_outputs.fix.md

```diff
@@ -0,0 +1 @@
+Remove warnings for unused outputs in `datadog_agent` source when the corresponding output is disabled in the source config.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** 🚨 grammar police

```suggestion
Removed warnings for unused outputs in `datadog_agent` source when the corresponding output is disabled in the source config.
```